index.html

```diff
@@ -1,304 +1,416 @@
-            <h2 class="headline-title2">I'm a Software Developer!</h2>
+  <body>
+    <main class="main-content">
+      <section class="header">
+        <nav class="menu-icon">
+          <h2><a href="index.html">My Logo</a></h2>
+          <nav class="dropdown-menu">
+            <img src="images/icons/Union.png" height="12px" width="18px" alt="Menu" />
+          </nav>
+          <div class="main-menu">
+            <ul>
+              <li>Portfolio</li>
+              <li>About</li>
+              <li>Contact</li>
+              <li><img src="images/icons/Icon-mail.svg" alt="Menu" /></li>
+            </ul>
+          </div>
+        </nav>
+      </section>
+
+      <section class="headline">
+        <section class="headline-body">
+          <div class="headline-body-column1">
+            <h1 class="headline-title">Hey There.<br />I'm Eric.</h1>
+            <h2 class="headline-title2">I'm a Software Developer</h2>
             <p class="headline-description">
-                I can help you build a product , feature or website Look through some of my work and experience! If you
-                like what you see and have a project you need coded, don’t hesitate to contact me.
+              I can help you build a product, feature or website. Look through
+              some of my work and experience! if you like what you see and have
+              a project you need coded, don't hesitate to contact me.
             </p>
-
-            <ul class="headline-social">
-                <li><a href="https://github.com/erickma1"><span class="social_icon git_hov"></span></a></li>
-                <li><a href="https://www.linkedin.com/in/eric-mawudeku-55b74883/"><span
-                            class="social_icon linked_hov"></span></a></li>
-                <li><a href="#"><span class="social_icon angel_hov"></span></a></li>
-                <li><a href="https://twitter.com/mawudeku"><span class="social_icon twitter_hov"></span></a></li>
-                <li><a href="#"><span class="social_icon med_hov"></span></a></li>
+          </div>
+          <div class="headline-body-column2">
+            <ul>
+              <li>
+                <a href="https://github.com/erickma1">
+                  <span class="icon git"></span>
+                </a>
+              </li>
+              <li>
+                <a href="https://www.linkedin.com/in/eric-mawudeku-55b74883/">
+                  <span class="icon linked"></span>
+                </a>
+              </li>
+              <li>
+                <a href="#">
+                  <span class="icon angel"></span>
+                </a>
+              </li>
+              <li>
+                <a href="https://twitter.com/mawudeku">
+                  <span class="icon twitter"></span>
+                </a>
+              </li>
+              <li>
+                <a href="#">
+                  <span class="icon medium"></span>
+                </a>
+              </li>
             </ul>
```

Comment:
- Good job done so far but as the previous reviewer mentioned, please make sure tat the color of the social media icons match the color in the design template. Please this 🙏 

Your project:
<img width="950" alt="y5yy" src="https://user-images.githubusercontent.com/73607512/236647541-aa6c5198-7d10-4778-8447-1f1b4e0a0f00.PNG">